crates/bevy_render/src/mesh/mesh.rs

```diff
@@ -453,20 +400,117 @@ pub mod shape {
             }
         }
     }
+
+    /// A Square prism. Just like a cube, but you can specify the dimensions.
+    pub struct SquarePrism {
```

Comment:
Can this be renamed `RectangularPrism`? A square prism would imply all sides are the same

Reply:
Good point!